core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/flatten.kt

```diff
@@ -9,37 +9,41 @@ import org.jetbrains.kotlinx.dataframe.columns.toColumnSet
 import org.jetbrains.kotlinx.dataframe.impl.api.flattenImpl
 import kotlin.reflect.KProperty
 
+public const val FLATTEN_DEFAULT_SEPARATOR: String = "_"
```

Comment:
It makes parameter info popup less informative. I think it's better to have it inlined 
![image](https://github.com/user-attachments/assets/c71ba5e7-0c94-4bb3-b003-8187705910cc)

![image](https://github.com/user-attachments/assets/539fd623-4b2e-4552-a2fe-e929547f1132)

![image](https://github.com/user-attachments/assets/6f5bf7e8-7629-49ea-8007-7c3eafb064bc)

Reply:
Come on IntelliJ XD. Don't make good coding practices harder

Reply:
There https://youtrack.jetbrains.com/issue/KTIJ-31662/Show-value-of-const-default-argument-when-calling-a-function

Reply:
Good c: Let's inline it before this issue is fixed?

Reply:
yes, a KDoc would also help, but it still doesn't show the value in the popup you show :/